src/main/java/net/atos/zac/app/productaanvragen/InboxProductaanvragenRESTService.java

```diff
@@ -79,9 +81,9 @@ public RESTResultaat<RESTInboxProductaanvraag> list(final RESTInboxProductaanvra
     }
 
     @GET
-    @Path("/{uuid}/aanvraagdocument")
-    public Response readFile(@PathParam("uuid") final UUID uuid) {
-        assertPolicy(policyService.readWerklijstRechten().getProductaanvragenInbox());
+    @Path("/{uuid}/pdfPreview")
+    public Response pdfPreview(@PathParam("uuid") final UUID uuid) {
```

Comment:
Hou de namen van methods in de typescript service en bijbehorende REST service altijd in sync zodat ze makkelijk te vinden zijn. Deze method heet in de typescript service (InboxProductaanvragenService) pdfPreviewURL. Hernoem deze of die in de typescript service.

Reply:
aangepast